tests/foreman/ui/test_contenthost.py

```diff
@@ -76,6 +79,16 @@ def setUpClass(cls):
             environment=cls.env,
             organization=cls.session_org,
         ).create()
+        if not settings.cdn:
+            # if not using cdn by settings, have to force uploading the
+            # manifest and attach a subscription to activation key
```

Comment:
@abalakh this setClass is not using force upload manifest and not using force cdn, we want simply that this was done here, we do not want to test all robottelo only for this TestCase, as we can implement regression